frontend/src/libs/components/sidebar/styles.module.scss

```diff
@@ -18,10 +18,13 @@
 }
 
 .sidebar {
+  position: sticky;
+  top: 0;
   display: flex;
   flex-direction: column;
   gap: 262px;
   width: 88px;
+  height: 100vh;
```

Comment:
We need to avoid vh, it may cause issues on mobile, try to use 100%

Reply:
@Liza-Veis it doesn`t work with 100%.
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92929504/3f8a0fad-568e-458a-9690-a871546bb060)

Reply:
for me it works without height property at all)

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/58943798/128a2876-5109-4a6d-a69d-dae67e277641)

Reply:
put smth big to have scroll

Reply:
now I see. Can we use `position: fixed` and `height: 100%`? It seems to be working

Reply:
if i change to position fixed i have to make margin left 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92929504/c37b318d-6859-4da1-88ca-96f54ab74dcd)

Reply:
you are right. But it is one of the working solutions. I've seen this on a lot of websites so I think this is not so bad

Reply:
@Liza-Veis what is the right way?

Reply:
will overflow-y hidden fix this (with position: sticky, height  100%) ? using vh for sidebar will create another issue later, that's why we can't use it

Reply:
@Liza-Veis height 100% doesn`t work. only vh works with position sticky. height 100% works with position fixed

Reply:
ok, got it, then let's use position fixed with padding. But pay attention padding should be in sidebar component, not in the app or somewhere else